### PR TITLE
Makes Bread Smite make Bong Bongs into bongbread

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request

What the title says. Essentially changes when somebody named “bong bong” is breaded into bongbread instead of regular bread

## Why It's Good For The Game

I think it is a neat addition and could be funny if this was the case.

## Changelog
:cl:
add: Added new things
/:cl:

### CODE REVIEW

File: code/modules/admin/smites/bread.dm [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
-/// Turns the target into bread
+/// Turns the target into bread (Bongbread if they are Bong Bong)
 /datum/smite/bread
 	name = "Bread"
 
 /datum/smite/bread/effect(client/user, mob/living/target)
 	. = ..()
-	var/mutable_appearance/bread_appearance = mutable_appearance('icons/obj/food/burgerbread.dmi', "bread")
+    if target.name == "Bong Bong"
```
Comment (Kitsunemitsu) [inline code/modules/admin/smites/bread.dm:9]: (target.name == "Bong Bong")

Need to bracket it chief

Comment (tile4) [reply code/modules/admin/smites/bread.dm]: Fixed

**Comment by Kitsunemitsu** [comment]
Christ, it's all broken
give me a bit to take a look

**Comment by Gboster-0** [comment]
its spaces on the added code instead of tabs, change it to tabs please

**Comment by tile4** [comment]
Sorry, was on my phone when I did this which didnt have a tab button in the built in keyboard as I know. So there. Will double check if I missed any but I think that's it

File: code/modules/admin/smites/bread.dm [hunk 2]
```diff
@@ -1,12 +1,15 @@
 #define BREADIFY_TIME (5 SECONDS)
 
-/// Turns the target into bread
+/// Turns the target into bread (Bongbread if they are Bong Bong)
 /datum/smite/bread
 	name = "Bread"
 
 /datum/smite/bread/effect(client/user, mob/living/target)
 	. = ..()
-	var/mutable_appearance/bread_appearance = mutable_appearance('icons/obj/food/burgerbread.dmi', "bread")
```
Comment (Gboster-0) [inline code/modules/admin/smites/bread.dm:9]: by the way, with how DM compiles you need to have var/mutable_appearance/bread_appearance always defined as a variable before code uses it, even if its defined by the if()else() check that always gives an output
in other words, keep the line above as
```
var/mutable_appearance/bread_appearance
``` and the code should be good

Comment (tile4) [reply code/modules/admin/smites/bread.dm:9]: So like that? Or did I misunderstand?

Comment (Gboster-0) [reply code/modules/admin/smites/bread.dm:9]: yep, but now you have to not re-define the same variable. So remove the other instances of `var/mutable_appearance/bread_appearance` replacing them with just `bread_appearance`

Comment (tile4) [reply code/modules/admin/smites/bread.dm:9]: Okay, there we go. Hope that is good?

Comment (Gboster-0) [reply code/modules/admin/smites/bread.dm:9]: Yep, should work perfectly well

**Pull request merged**